src/dotnet/Fable.Tools/ProjectCracker.fs

```diff
@@ -190,18 +190,29 @@ let crackFsproj (projFile: string) =
         |> List.filter (fun x -> not(x.EndsWith("Fable.Core.dll")))
         |> List.map (fun x -> Path.Combine(projDir, Path.normalizePath x) |> Path.GetFullPath) }
 
-let rec findPaketDependenciesDir dir =
-    if File.Exists(Path.Combine(dir, "paket.dependencies")) then
+let rec findPaketDependenciesDir dir searchedDirs = 
+    let path = Path.Combine(dir, "paket.dependencies")
+    if File.Exists(path) then
+        Log.print false "Found %s inside %s" path dir
         dir
     else
         match Directory.GetParent(dir) with
-        | null -> failwith "Couldn't find paket.dependencies directory"
-        | parent -> findPaketDependenciesDir parent.FullName
+        | null -> 
+            searchedDirs 
+            |> String.concat "\n" 
+            |> failwithf "Couldn't find paket.dependencies directory, searched in: \n%s"
+        | parent -> 
+            let searched = dir :: searchedDirs
+            findPaketDependenciesDir parent.FullName searched
 
 let tryFindPaketDirFromProject projFile =
     let projDir = Path.GetDirectoryName(projFile)
     if File.Exists(Path.Combine(projDir, "paket.references"))
-    then findPaketDependenciesDir projDir |> Some
+    then 
+        try 
+            findPaketDependenciesDir projDir [] |> Some
+        with
+        | _ -> None
```

Comment:
@alfonsogarciacaro Confirmed that the try-catch is not needed. My project compiled fine